src/main/scala/se/nimsa/sbx/storage/StorageProtocol.scala

```diff
@@ -29,7 +28,7 @@ object StorageProtocol {
     minimumPixelValue: Int,
     maximumPixelValue: Int)
 
-  case class TagMapping(tagPath: TagPathTag, value: ByteString)
```

Comment:
Technically a breaking change, but the format format with base64-encoded bytes was useless. I don't think anyone has put it to good use.